app/services/tools/base.py

```
@@ -262,6 +278,12 @@ async def is_ready(self) -> bool:
         # an optional binary override this (see nsz / prod.keys).
         return True
 
+    def expected_output_size(self, input_path: str, mode: str) -> int | None:
```

Ruff (0.16.1)

[warning] Unused method argument: `input_path`

(ARG002)